src/components/Elements/MenuItem.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+  * Renders a menu item using router links. Image is optional.
+  */
+function MenuItem(props) {
```

Comment:
nit: we can pull out the various properties here for better readability in the jsx syntax. i.e.
`const {onClick, imageType, title} = this.props;`

and then you can just use `onClick` `imageType` and `title` in the jsx.